lib/spack/spack/container/images.py

```diff
@@ -50,9 +50,6 @@ def build_info(image, spack_version):
     if not build_image:
         return None, None
 
-    # Translate version from git to docker if necessary
-    build_tag = image_data["build_tags"].get(spack_version, spack_version)
-
     return build_image, build_tag
```

Comment:
If I read correctly from the web UI, here you are returning `build_tag` without ever defining it. Should probably be:
```suggestion
    return build_image, spack_version
```